lib/auth/mocku2f/mocku2f.go

```diff
@@ -187,8 +207,14 @@ func (muk *Key) signRegister(appIDHash, clientDataHash []byte) (*signRegisterRes
 		return nil, trace.Wrap(err)
 	}
 
+	var flags = uint8(u2fRegistrationFlags)
+	if muk.SetUV {
+		// Mimic WebAuthn flags if SetUV is true.
+		flags = uint8(upBit | uvBit | atBit)
```

Comment:
Do you intend to overwrite `u2fRegistrationFlags` here? Intuitively I kind of expected an `|=`.

Reply:
I did. It's one of the cases where the standards are just different in terms of intent, `0x05` is a reserved value in CTAP1.2, it doesn't mean anything like UV or AT.

Reply:
Isn't `u2fRegistrationFlags` just `upBit | uvBit`?

Reply:
Numerically yes, but semantically no. Verification is not guaranteed by U2F.